applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -35,32 +35,49 @@ enum state {
 
 static enum state module_state = STATE_DISABLED;
 
+#define LIST_FOR_DVFS_STATE(MACRO, ...)						\
+	MACRO(INITIALIZING, __VA_ARGS__)					\
+	MACRO(LLPM_CONNECTED, __VA_ARGS__)					\
+	MACRO(USB_CONNECTED, __VA_ARGS__)					\
+	MACRO(CONFIG_CHANNEL, __VA_ARGS__)					\
+	MACRO(SMP_TRANSFER, __VA_ARGS__)					\
+	MACRO(COUNT, __VA_ARGS__)
+
+#define DVFS_STATE(name, ...) DVFS_STATE_ ## name,
+
+#define GET_ENUM_STRING(name, ...)						\
+	IF_ENABLED(CONFIG_DESKTOP_DVFS_STATE_ ## name ## _ENABLE,		\
+		(case DVFS_STATE_ ## name: return STRINGIFY(_CONCAT(DVFS_STATE_, name));))
```

Comment:
Can we reuse `DVFS_STATE(name, ...)` macro? Also in other places

Reply:
No, as it has "," sign see: https://github.com/nrfconnect/sdk-nrf/pull/16899#discussion_r1730783469